tests/e2e/cluster_downgrade_test.go

```diff
@@ -92,12 +92,25 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int, triggerSnapshot bool) {
 			Storage: currentVersionStr,
 		})
 	}
+	cc := epc.Etcdctl()
 	t.Logf("Cluster created")
+	if len(epc.Procs) > 1 {
+		t.Log("Waiting health interval to required to make membership changes")
+		time.Sleep(etcdserver.HealthInterval)
+	}
+
+	t.Log("Adding member to test membership, but a learner avoid breaking quorum")
+	resp, err := cc.MemberAddAsLearner(context.Background(), "fake1", []string{"http://127.0.0.1:1001"})
+	require.NoError(t, err)
 	if triggerSnapshot {
 		t.Logf("Generating snapshot")
-		generateSnapshot(t, snapshotCount, epc)
+		generateSnapshot(t, snapshotCount, cc)
+		verifySnapshot(t, epc)
```

Comment:
Suggest to verify the snapshot using `verifySnapshotMembers`. 

FYI. https://github.com/etcd-io/etcd/pull/16376

and also https://github.com/etcd-io/etcd/pull/16458

Reply:
Don't think this is needed. Here this is a downgrade/upgrade test, so from downgrade perspective we just need to confirm that members don't change during downgrade/upgrade procedure. Checking exact list of members is not in scope of the test. Aka, I would treat members as black box instead of whitebox.